test/end-to-end/Azure.Functions.NodejsWorker.E2E/Azure.Functions.NodejsWorker.E2E/HttpEndToEndTests.cs

```diff
@@ -72,6 +72,15 @@ public async Task HttpTriggerWithCookieTests()
             // Assert.Equal("mycookie4-samesite-none=myvalue; path=/; samesite=none", cookies[5]);
         }
 
+        [Fact]
+        public async Task HttpTriggerBindingDataTests()
+        {
+            HttpResponseMessage response = await HttpHelpers.InvokeHttpTrigger("HttpTriggerBindingData");
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            string responseBody = await response.Content.ReadAsStringAsync();
```

Comment:
Possible to add a test that uses binary data?

Reply:
It doesn't quite look like it, but this is testing that :) https://github.com/Azure/azure-functions-nodejs-worker/blob/dev/test/end-to-end/Azure.Functions.NodejsWorker.E2E/Azure.Functions.NodejsWorker.E2E/HttpEndToEndTests.cs#L48